src/System.Net.HttpListener/tests/HttpListenerPrefixCollectionTests.cs

```diff
@@ -216,7 +216,7 @@ public void Add_PrefixAlreadyRegisteredAndStarted_ThrowsHttpListenerException()
 
         [Theory]
         [InlineData("http://microsoft.com/")]
-        [InlineData("http://[]/")]
+        //Managed implementation allows this. Is this a bad test? [InlineData("http://[]/")]
```

Comment:
@hughbe, for now, let's split it into a separate test that's Windows-specific, e.g.
```C#
[ActiveIssue(..., TestPlatforms.AnyUnix)] // Fails by design on Windows but is allowed by the managed implementation
[Theory]
[InlineData("http://[]/")]
public void Add_InvalidPrefixNotStarted_ThrowsHttpListenerExceptionOnStart_Windows(string uriPrefx) =>
    Add_InvalidPrefixNotStarted_ThrowsHttpListenerExceptionOnStart(uriPrefix);
```
and we can follow-up on it separately. No need to hold up all of these changes for this corner case.